business/mesh.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/kiali/kiali/util/httputil"
 )
 
+const DefaultClusterID = "Kubernetes"
```

Comment:
See my comment in the backport PR (i.e. should this be all lowercase?)

Reply:
It is correct as is.  It mirrors the default set by Istio.